services/core-web/src/components/dashboard/majorProjectHomePage/MajorProjectTable.tsx

```diff
@@ -0,0 +1,227 @@
+                    .map((detail) =>
+                      mineCommodityHash ? mineCommodityHash[detail.mine_commodity_code] : ""
+                    )
+                );
+              }
+              return result;
+            }, [])
+          )
+        )
+        : [],
+    update_timestamp: formatDate(project.update_timestamp),
+    project_summary: project.project_summary,
+    information_requirements: project.information_requirements,
+    major_mine_application: project.major_mine_application,
+    project_decision_package: project.project_decision_package,
+  }));
+
+const transformChildRowData = (section, project) => {
+  return {
+    key: section.key,
+    tab: section.tab,
+    projectGuid: project.key,
+    status: statusCodes[section.status_code] || statusCodes["NTS"],
+    updateTimestamp: formatDate(section.update_timestamp) || Strings.EMPTY_FIELD,
+    updateUser: section.update_user,
+  }
+}
+
+const handleTableChange = (handleSearch, tableFilters) => (pagination, filters, sorter) => {
+  const params = {
+    ...tableFilters,
+    sort_field: sorter.order ? sorter.field : undefined,
+    sort_dir: sorter.order ? sorter.order.replace("end", "") : undefined,
+  };
+  handleSearch(params);
+};
+
+export const MajorProjectTable: FC<MajorProjectsSearchFormProps> = ({
+  projects,
+  mineCommodityOptionsHash,
+  handleSearch,
+  filters,
+  isLoaded,
+  sortField,
+  sortDir,
+  expandedRowKeys,
+  onExpand,
+}) => {
+  const { isFeatureEnabled } = useFeatureFlag();
+  const projectDecisionPackageEnabled = isFeatureEnabled(Feature.MAJOR_PROJECT_DECISION_PACKAGE);
+  const columns = [
+    {
+      key: "project_title",
+      title: "Project Title",
+      dataIndex: "project_title",
+      sortField: "project_title",
+      sorter: true,
+      render: (text, record) => (
+        <Link to={router.EDIT_PROJECT.dynamicRoute(record.key)} title="Name">
+          {text}
+        </Link>
+      ),
+    },
+    renderTextColumn("project_id", "Client ID", true),
+    renderTextColumn("mine_name", "Mine Name", true),
+    renderTextColumn("project_lead_name", "EMLI Project Lead", true),
+    {
+      key: "commodity",
+      title: "Commodity",
+      dataIndex: "commodity",
+      sortField: "commodity",
+      sorter: false,
+      render: (text) => (
+        <div title="Commodity">
+          {(text && text.length > 0 && text.join(", ")) || Strings.EMPTY_FIELD}
+        </div>
+      ),
+    },
+    {
+      title: "Updated Date",
+      key: "update_timestamp",
+      dataIndex: "update_timestamp",
+      sortField: "update_timestamp",
+      render: (text) => <div title="Updated Date">{text}</div>,
+      sorter: dateSorter("update_timestamp"),
+      defaultSortOrder: "descend",
+    },
+    {
+      title: "",
+      key: "project",
+      dataIndex: "project",
+      render: (text, record) => (
+        // @ts-ignore
+        <div title="" align="right" data-cy="major-projects-table-open-button">
```

Comment:
does the `align="right"` even work on div? Or does it just want to be corrected to `style={{align:"right"}}`?